addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -1,10 +1,101 @@
 odoo.define('mass_mailing.snippets.options', function (require) {
 "use strict";
 
-var options = require('web_editor.snippets.options');
+const options = require('web_editor.snippets.options');
 const {ColorpickerWidget} = require('web.Colorpicker');
+const SelectUserValueWidget = options.userValueWidgetsRegistry['we-select'];
+const weUtils = require('web_editor.utils');
 const {_t} = require('web.core');
 
+//--------------------------------------------------------------------------
+// Constants
+//--------------------------------------------------------------------------
+
+const _getFontName = fontFamily => fontFamily.split(',')[0].replace(/"/g, '').replace(/([a-z])([A-Z])/g, (v, a, b) => `${a} ${b}`).trim();
```

Comment:
Well that's true. I like it like this since it's declared at the top of the file and used further down. That way we can't confuse it for an util that might have been imported.